Simplenote/src/main/java/com/automattic/simplenote/PreferencesFragment.java

```diff
@@ -475,17 +476,36 @@ public int compare(String text1, String text2) {
 
     private void importData(Uri uri) {
         try {
-            Importer.fromUri(this, uri);
+            AppLog.add(Type.IMPORT, "Importing notes from " + uri + ".");
+
+            FragmentActivity activity = getActivity();
+            if (activity == null) {
```

Comment:
Question (❓): Although this case will not happen that ofter, when or if it does, shouldn't we show a `toast` of some kind, just it happens when an exception is thrown? This way the user will know what happen. Otherwise, this code branch will silently complete without any message, leaving the user confused. Wdyt?

Reply:
👋  @ParaskP7. Yes, this is an edge case that will rarely happen. When it happens, is because some weird callback interleaving created by either device rotation or closing the app. The problem of showing a `Toast` here is that if the activity is `null` for the fragment, how we obtain the application context to show the `Toast`. Do you know if `requireContext()` does not throw an exception if activity is `null`?

Reply:
👋 @danilo04 !

Actually, I was overthinking it and you're right, when and if that happens, there are going to be bigger problems than just showing a toast, at which case I am now thinking that logging would be enough so that we at least know that this happened. Please ignore my comment, I think we are good if we leave that as is for now.

🙇